students/Ajay_Randhawa/lesson02/Grid_printer2.py

```diff
@@ -0,0 +1,14 @@
+def print_grid2(s, n):
+	plus = "+"
```

Comment:
Over indented - make sure you have your text editor set up to indent 4 spaces (no tabs)